CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Visit https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+# for learning about the syntax
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @rasswanth-s @aanurraj @gmuraru @kamathhrishi
```

Comment:
Could you have the global owner order like:
@gmuraru @aanurraj  @kamathhrishi  @rasswanth-s

Reply:
Sure!

Reply:
Ahh..it was also ok like that! :smile: